src/RefractAPI.cc

```diff
@@ -38,6 +38,26 @@ namespace drafter {
             return ptr == NULL;
         }
 
+        void RemoveEmptyElements(RefractElements& elements)
+        {
+            elements.erase(std::remove_if(elements.begin(), elements.end(), IsNull<refract::IElement>), elements.end());
+        }
+
+        refract::IElement* DataStructureToRefract(const snowcrash::DataStructure& dataStructure)
+        {
+            refract::IElement* ds = MSONToRefract(dataStructure);
```

Comment:
Please use `dataSturctureElement` instead of `ds`. The variable names should be understandable.